src/tagger/imgurClient.js

```diff
@@ -0,0 +1,35 @@
+const imgur = require('imgur')
+
+const TaggerDatabaseManager = require('./TagDatabaseManager')
+
+class ImgurClient {
+  constructor () {
+    this.tdbm = new TaggerDatabaseManager()
+
+    imgur.setClientId(process.env.IMGUR_CLIENT_ID)
+    imgur.setAPIUrl('https://api.imgur.com/3/')
+  }
+
+  async upload (id, imgObj) {
+    const { data } = await imgur.uploadUrl(imgObj.image)
+
+    try {
```

Comment:
don't need async await here we can just return the promise and use .then and .catch